stdlib/_ast.pyi

```diff
@@ -1049,13 +1628,32 @@ if sys.version_info >= (3, 10):
         else:
             def __init__(self, subject: expr, cases: list[match_case], **kwargs: Unpack[_Attributes]) -> None: ...
 
+        if sys.version_info >= (3, 14):
+            def __replace__(
+                self,
+                *,
+                subject: expr,
+                cases: list[match_case],
+                **kwargs: Unpack[_Attributes],
+            ) -> Self: ...
+
     class pattern(AST):
         lineno: int
         col_offset: int
         end_lineno: int
         end_col_offset: int
         def __init__(self, **kwargs: Unpack[_Attributes[int]]) -> None: ...
 
+        if sys.version_info >= (3, 14):
+            def __replace__(
+                self,
+                *,
+                lineno: int,
+                col_offset: int,
+                end_lineno: int,
+                end_col_offset: int,
+            ) -> Self
```

Comment:
```suggestion
            ) -> Self: ...
```

Reply:
Going to push a commit fixing this and adding the defaults.